site/docs/plugins/guide.md

```diff
@@ -1,5 +1,5 @@
 ---
-prev: /plugins/
+prev: ./
```

Comment:
Have you checked that relative paths work here?

Reply:
It's working fine.

Reply:
Neat!

Reply:
Oh, it should be `../plugin`. `./` can only work in EN docs. Fixed in commit https://github.com/grammyjs/website/pull/434/commits/1169d2426112ad5afba5becbc356fc7dd43bd380.